include/experimental/group/gemm/impl/int4_dequantize_xmx_xe.hpp

```diff
@@ -508,14 +506,14 @@ class gemm_t<compute_policy_int4_dequantize_xmx<compute_attr_,
 
                 //2: int8 includes 2 4bits data.
                 xetla_vector<uint8_t, block_size_x_b * block_size_y_b> cvt_blk;
-                cvt_blk.xetla_select<matB_t::block_elems, 2>(0)
-                        = matB_blk & 0x0f;
-                cvt_blk.xetla_select<matB_t::block_elems, 2>(1) = matB_blk >> 4;
-
```

Comment:
& 0x0f and  >> 4
For the two quant methods, it seems that they can be reused

Reply:
no, one stored in cvt_blk, another in cvt_blk_i8.